apps/src/code-studio/components/aiTutor/aiTutorPanelContainer.jsx

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import {tryGetLocalStorage, trySetLocalStorage} from '@cdo/apps/utils';
+
+export default class AITutorPanelContainer extends React.Component {
+  static propTypes = {
+    children: PropTypes.node,
+  };
+
+  state = {open: tryGetLocalStorage('ai-tutor-panel', 'open') !== 'closed'};
```

Comment:
What do we gain by using local storage here instead of having it be a boolean on local state?

Reply:
Works for me, especially since this is prototype UI.